rook/toolbox-deploy.yaml

```diff
@@ -37,10 +43,6 @@ spec:
               name: ceph-config
             - name: mon-endpoint-volume
               mountPath: /etc/rook
-          resources:
-            requests:
-             cpu: "5m"
-             memory: "10Mi"
```

Comment:
Why remove resource request? This doesn't follow our best practice. I would rather add limits to it as well.

Reply:
Okay, so you argue that we need to depart a bit from the official toolbox manifest, right?

Or should we upstream the `resources`?

Reply:
But otherwise I don't see an issue with deviating in this case since this is just a single manifest tool. So I don't see any burden to manage these small changes.